example/40-validatingwebhookconfiguration.yaml

```diff
@@ -26,3 +26,26 @@ webhooks:
+- name: secrets.validation.alicloud.provider.extensions.gardener.cloud
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - UPDATE
+    resources:
+    - secrets
+  failurePolicy: Fail
+  objectSelector: {}
+  namespaceSelector: {}
+  sideEffects: None
+  admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      namespace: garden
+      name: gardener-extension-admission-alicloud
+      path: /webhooks/validate/secrets
+    caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMrekNDQWVPZ0F3SUJBZ0lSQUppUXlMcC90N2ErcjlLaFRNd2lNRlV3RFFZSktvWklodmNOQVFFTEJRQXcKRnpFVk1CTUdBMVVFQXhNTVoyRnlaR1Z1WlhJdFpHVjJNQjRYRFRJeE1ERXlNakV5TURBek5Gb1hEVE14TURFeQpNakV5TURBek5Gb3dGekVWTUJNR0ExVUVBeE1NWjJGeVpHVnVaWEl0WkdWMk1JSUJJakFOQmdrcWhraUc5dzBCCkFRRUZBQU9DQVE4QU1JSUJDZ0tDQVFFQTVJRVk0RysrOFEvWDc2dExZRWhjbzhmeXRsSmtqMmUyU3JheGRVMzAKTjJlZEdZS0Qzbi85VXY4SFZhMGNsTVJ2WVdsSGhUeDh6Qkt0TWxFZmJFZVRNQXZ0MUw2bmZIQmNKemNiUUlEZwpIbUg1SVRpYVRtUitlNGh2WXZxbjZhNnJQZ0wxa0NVYlFRWWJLQjlTSEdqb1M0UlFjQWQ1b0ZRQ3EvUDdrQ2N2CldkZWowTkMreFlmQk80Q0dkVWRGV09hM1UxZ2JQT0t6eW8wUFhNN3F1eVNYRzQ1UG15dTV3S3RNYTdzVE5UUGMKanNCVTdsVW9mYkhqaTJaaVFpSVR1V09Pc3RLbjJDQXh3QlNEVnI2OFlGbTVGSHZickZ0ZGlOUVpQQ0l3MDhYaApubVl1enhMbHg4NCtackt3RTBMQ3JmTmMxTkYwRmFYOWYrdnVmM3RMWmZwekhRSURBUUFCbzBJd1FEQU9CZ05WCkhROEJBZjhFQkFNQ0FhWXdEd1lEVlIwVEFRSC9CQVV3QXdFQi96QWRCZ05WSFE0RUZnUVVXV0tpdiswZTNQVU0KbHpJMnNHdTlIUUxLZzBnd0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFCL0o0YVdYMm9PaFY3MkxoTkt2YUpIdQpSUmVpcHFRbXAwQTA1UE1WODJ0U3g1RUtzV0hCcWxkUUlrcmMwYXpiZDRHK1dqTDMzeFJCelYvT3BlZmp4ZzN3ClhyWElPcWlCMUxwTVBjNmdkNndvRWp6R2cvalVJd0xCcjROY2tiWktZMU4rQ3FOQXRQYUc2b0JXaXlIZ3cwUFcKaElXNU5BeDQvWUQ4QnVHYzRKTi9CSldRbTZOcWhQVzRRQWVFaU9VVG5JU2JyajM2c2paRmo1VHY3OTcvRjBqSApWakUxQVpNbTRjUHU1bkF2VlFTdCtyV0hQVW1ac3cxZnJQNWRNdTV2cXM5Y2JuT3VhNlUxQzNWRjZoZXNDbm1LCkhWNXNYYThiaU03MWNJVys0bkhpYVlmZHFraFBUZ1pXSlVLeFE3NCtCNWQ1U0ZiekRkTmkvT1R3T1NxMDBQOD0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
This `clientConfig` uses a `service`, while the one above (for other resources than secrets) uses a url. Eventually though they will both resolve to e.g. `https://<ip>:9443/webhooks/validate/*`. Could they be both `url` or both `service`, is there any special reason to keep them different?

Reply:
See https://github.com/gardener/gardener-extension-provider-gcp/pull/112#discussion_r447654478